my.css

```diff
@@ -0,0 +1,104 @@
+.header {
+	position: relative;
+	height: 100px;
+	background: -webkit-linear-gradient(180deg, white 63%, black 37%);
+}
+.burgermenu {
+	position: absolute;
+	right: 0;
+	top: 0;
+	width: 70px;
+	height: 70px;
+	background: gray;
+}
+.burger {
+	position: absolute;
+	width: 30px;
+    height: 6px;
+    margin-left: 22px;
+    margin-top: 22px;
+    background: #000;
+}
+.burger::before {
+	content: '';
+	position: absolute;
+	width: 30px;
+    height: 6px;
+    margin-left: 1px;
+    margin-top: 16px;
+    background: #000;
+}
+.header__promo {
+	width: 35%;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	font-size: 25px;
+}
+.logo__promo {
+	color: #fff;
+	padding-top: 20px;
+    padding-left: 20px;
+}
+.ru__link {
+	color: #fff;
+	padding-top: 50px;
+    padding-left: 10px;
+    text-decoration: none;
+}
+.main {
+   position: relative;
+   width: 100%;
+   height: 800px;
+   background: -webkit-linear-gradient(180deg, white 63%, black 37%);
+}
+.collection {
+	display: flex;
+	justify-content: center;
+}
+.newcollection {
+	display: flex;
+	flex-direction: column;
+	width: 30%;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5556249)